deeppavlov/models/morpho_tagger/network.py

```diff
@@ -13,16 +13,19 @@
 # limitations under the License.
 
 from logging import getLogger
-from typing import List, Union, Tuple, Iterable
+from typing import List, Union, Tuple, Iterable, Optional
+from copy import deepcopy
+import inspect
```

Comment:
It's better to put `import <module>` clauses in alphabetical order before `from <module> import <symbol>` in alphabetical order in every import group. So it should look like
```python3
import inspect
from copy import deepcopy
from logging import getLogger
from typing import List, Union, Tuple, Iterable, Optional
```